### PR TITLE
fix(obligations): fix UI and connected db to dropdown menu

### LINKED ISSUE #1300: Dropdowns in obligation admin UI are to small to display actual text
### Description

In the "Add Obligation"-admin UI are the drop-down elements to small to actually display the current state. They just show `...`.

### Versions

* Last commit id on master: https://github.com/fossology/fossology/commit/69629cc1f45f2cbde7e1318dba8cf96a3ac3de8a
* Operating System: Linux
* Browser: Chromium (works in Firefox)

### PR DESCRIPTION
**Pull request by hastagAB**
Fix #1300  & #1306 

# Description :
The smaller size of dropdown UI (Showing Three-dots earlier) was fixed. 
The database was not connected to the Dropdown menu so it was showing default selection while editing. [fixed]

# Example Screenshot : 
new1

### CODE REVIEW

**Comment by GMishx** [comment]
Thanks for the contribution. Please follow the [Contribution guidelines](https://github.com/fossology/fossology/blob/master/CONTRIBUTING.md) while creating a commit.

**Comment by mcjaeger** [comment]
right, the commit message should be like `fix(obligations): fix UI and connected db to dropdown menu` ... it is just the conventional changelog

**Comment by hastagAB** [comment]
Sorry for the Delay! I was in the middle of my exams.
Fixed the commit message according to the contribution guidelines.
Hope this is fine now.

File: src/www/ui/admin-obligation-file.php [hunk 1]
```diff
@@ -262,6 +262,8 @@ function ObligationTopic($topic)
       $ob .= "<td align=center>$associatedLicenses</td>";
       $ob .= "<td align=center>$candidateLicenses</td>";
       $ob .= "</tr>";
+      $ob_type = $row['ob_type'];
+      $ob_classification = $row['ob_classification'];
```
Comment (GMishx) [inline src/www/ui/admin-obligation-file.php]: Can you please describe the use of these two new variables? They seems to be unused.

Comment (hastagAB) [reply src/www/ui/admin-obligation-file.php]: The variables are used in src/www/ui/template/admin_obligation-upload_form.html.twig. Line 28 and 45.
These variables are the actual Obligation Type (ob_type) and Obligation Classification (ob_classification) related to that particular obligation fetched from the database

Comment (hastagAB) [reply src/www/ui/admin-obligation-file.php]: Earlier these two parameters were hardcoded as string values on line 28 and 45 as 'Obligation' and 'green' respectively

Comment (GMishx) [reply src/www/ui/admin-obligation-file.php]: The values in the `*.twig` templates goes in the form of array keys. If you want to pass some value to the twig file, you should add it to any one of the array near https://github.com/fossology/fossology/blob/master/src/www/ui/admin-obligation-file.php#L365 as it is done in https://github.com/fossology/fossology/blob/master/src/www/ui/admin-obligation-file.php#L338.

Comment (GMishx) [reply src/www/ui/admin-obligation-file.php]: Hello @hastagAB, are you planning to working on this?

Comment (hastagAB) [reply src/www/ui/admin-obligation-file.php]: The selectsingle twig function on line 28 and 45 in the `admin_obligation-upload_form.html.twig` file is expecting a selected value associated with that particular Obligation. But in the current version of the code, the function is being passed a static string value. 
In line 28 it is currently 'Obligation' so no matter what the actual value is the dropdown will always show 'Obligation'
In line 45 it is currently 'green' so no matter what the actual value is in the database the dropdown will always show 'green'

I have replaced these two static strings with the corresponding variables defined in `admin-obligation-file.php` which stores the actual value associated with the particular Obligation.

Comment (GMishx) [reply src/www/ui/admin-obligation-file.php]: I did understood your point, what I was trying to convey is that the twig needs input for placeholders as an index in an array before rendering the twig template. You can check the code lines I've share before.

**Pull request merged**